README.md

```diff
@@ -77,6 +77,20 @@ MtBatchTextSmsResult batch = conn.createBatch(
 System.out.println("Successfully sent batch " + batch.id());
 ```
 
+#### Sending MMS Message
+
+```java
+String sender = "SENDER"; // Optional, must be valid phone number, short code or alphanumeric.
+String [] recipients = {"1232323131", "3213123"};
+MtBatchMmsResult batch =
+conn.createBatch(
+    SinchSMSApi.batchMms()
+        .sender(sender)
+        .addRecipient(recipients)
+        .body(SinchSMSApi.mediaBody().url("http://test.url.com").build())
```

Comment:
I wonder if we should put in the message text in the example as well even though it is optional. What do you think?

Reply:
Sure, will be good to add